controllers/User/userController.js

```diff
@@ -0,0 +1,82 @@
+const validateUser = require('../../utils/validations/index')
+const bcrypt = require('bcryptjs')
+const User = require('../../models/User/userModel')
+const response = require('../../utils/libs/response')
+const Joi = require('joi')
+
+// Register Validation schema
+const registrationSchema = Joi.object({
+    firstName: Joi.string().required(), 
+    lastName: Joi.string().required(), 
+    email: Joi.string().email().required(),
+    password: Joi.string().required().min(8),
+    phone: Joi.number().required()
+})
+
+// login Validation schema
+const loginSchema = Joi.object({
+    email: Joi.string().email().required(),
+    password: Joi.string().required().min(8)
+})
```

Comment:
Please move the joi validation schema to the utils folder, you create a folder called validations inside the utils and then create a js file and add those joi validation.

Reply:
okay boss